### ISSUE TITLE
Failed to complete TLS handshake: unexpected end of file

### ISSUE THREAD

**Issue by rodael** [CM000n/qss#15]
**Describe the bug**
Fresh QuestDB install (docker), initial load produces this log entry in home assistant:

``` 
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/tenacity/__init__.py", line 382, in __call__
    result = fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/qss/io.py", line 43, in _retry_data_insertion
    _insert_row(host, port, auth, event)
  File "/config/custom_components/qss/io.py", line 17, in _insert_row
    with Sender(host, port, auth=auth, tls=tls) as sender:
  File "src/questdb/ingress.pyx", line 1530, in questdb.ingress.Sender.__enter__
  File "src/questdb/ingress.pyx", line 1520, in questdb.ingress.Sender.connect
questdb.ingress.IngressError: Failed to complete TLS handshake: unexpected end of file

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/threading.py", line 1045, in _bootstrap_inner
    self.run()
  File "/config/custom_components/qss/__init__.py", line 156, in run
    insert_event_data_into_questdb(self.host, self.port, self.auth, event, self.queue)
  File "/config/custom_components/qss/io.py", line 48, in insert_event_data_into_questdb
    _retry_data_insertion(host, port, auth, event)
  File "/usr/local/lib/python3.11/site-packages/tenacity/__init__.py", line 289, in wrapped_f
    return self(f, *args, **kw)
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/tenacity/__init__.py", line 379, in __call__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/tenacity/__init__.py", line 326, in iter
    raise retry_exc from fut.exception()
tenacity.RetryError: RetryError[<Future at 0x7f7dcd0b45d0 state=finished raised IngressError>]
```

**To Reproduce**

Run QuestDB with docker-compose:

```
version: '2.4'
services:
    questdb:
        container_name: questdb
        restart: unless-stopped
        image: questdb/questdb:latest
        ports:
            - 9000:9000 # REST/CONSOLE
            - 8812:8812 # POSTGRES-WIRE
            - 9009:9009 # INFLUXDB-LINE
        volumes:
          - questdb:/var/lib/questdb

volumes:
  questdb:
```

Add qss and configure it like this (obviously replace the host):

```
qss:
  host: "10.0.0.17"
  port: 9009
```

**Expected behavior**
Create the table and insert data

**Comment by rodael** [comment]
I'm not too familiar with python, but it works if I remove the auth and tls part from line 17 in io.py:

from
```
    with Sender(host, port, auth=auth, tls=tls) as sender:
```
to
```
    with Sender(host, port) as sender:
```

So something is off with the tls/auth variable

**Comment by saintman23** [comment]
I've got the same error after update to latest qss version on HA 2023.12.2

**Comment by CM000n** [comment]
Yes, it seems that the standardisation of the two transmission variants with and without TLS has led to this error. I will look at it again soon. Thanks for reporting it.

**Comment by CM000n** [comment]
Should be fixed with 0.0.8
